pkg/apis/kops/networking.go

```diff
@@ -126,3 +127,14 @@ type CiliumNetworkingSpec struct {
 	Debug  bool   `json:"debug,omitempty"`
 	Tunnel string `json:"tunnel,omitempty"`
 }
+
+// LyftIpVlanNetworkingSpec declares that we want to use the cni-ipvlan-vpc-k8s CNI networking
+type AmazonVPCIPVlanNetworkingSpec struct {
+	// Tags that subnets must have to be used for pod IPs
+	SubnetTags      map[string]string `json:"subnetTags,omitempty"`
```

Comment:
So do we need all of these in order to start trying it out?  I'm not sure how far along you are, but the challenge is that once we put them in the API, they're part of our contract.  Can we default these or live without them, at least for now?

Reply:
SecurityGroupIds is  set to the sgs on the primary ENI if nothing overrides it so we could just move that into a template function and not allow overrides to start
InterfaceIndex can just be 1. It's the suggested way of doing it 
RouteToVPCPeers could just be true. I imagine there are situations where people want to not have this turned on
The SubnetTags could just do Type = pod. Then in the future maybe if kops creates subnets for the pod network it could add this tag to them.
BinariesDownloadURL will get removed altogether since it's handled as an asset now.